examples/custom_raw.py

```diff
@@ -11,8 +11,8 @@
 
 
 class MyRawChecker(BaseRawFileChecker):
-    """Check for line continuations with '\' instead of using triple
-    quoted string or parenthesis
+    r"""Check for line continuations with '\' instead of using triple
```

Comment:
Ironic update to use raw docstring in the MyRawChecker class 😄

Reply:
Did you know it was twice voted "most likely place to see dangerous raw characters in the doc" ?

Reply:
No :) what is that (doc)?

Reply:
Just joking 😄 The doc would be the docstring in the raw checker where you explain about raw string and why you need them sometime.